### ISSUE TITLE
Commands Print and Execute Together

### ISSUE THREAD

**Issue by suhaildawood** [rgardler/simdem#24]
As a result of #23, SimDem now prints and executes commands together, as opposed to first printing the command, waiting for a key press, then executing the command.

**Comment by rgardler** [comment]
This was a deliberate change.One less click to move forwards. Do you feel it is a mistake?

**Comment by suhaildawood** [comment]
Oh I see! To make it seem more realistic, a delay could be added between the completion of 'typing' and executing the command.

**Comment by rgardler** [comment]
Yes, that might work. Perhaps a last keypress delay in the typing simulation. More generally it may be a good idea to make this configurable. I can imagine some people wanting  it to pause for a keypress on each command.
